disputer/disputer.js

```diff
@@ -86,7 +86,7 @@ class Disputer {
       } catch (error) {
         Logger.error({
           at: "Disputer",
-          message: `Failed to dispute liquidation`,
+          message: "Failed to dispute liquidation",
```

Comment:
These code changes got through the linter before due to `--no-verify` pushes

Reply:
hmmmm, but the linter should check *everything*...? husky should be a totally optional thing IIUC. Like we're not depending on it to catch errors that ci won't catch, right?

I'm assuming the explanation is that we didn't add this part of the repo to the prettier command (which was a mistake).

Reply:
I thought so. I'm not sure how this go in but the linter in CI didnt catch it. You are right that husky is totally optional; if you choose to not use and and push anyways with incorrectly formatted code CI should throw it out from `lint.sh`. 

I think your explanation makes sense but it was specified in our `package.json` 🤔